src/components/Person.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Person(props) {
```

Comment:
```suggestion
function Person({
  name,
  age,
  sex,
  isMarried,
  partnerName
}) {
```
You could use destructuring